keyboards/mincedshon/ecila/config.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 mincedshon (@mincedshon)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x534B
+#define PRODUCT_ID      0x534B
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    mincedshon
+#define PRODUCT         ecila
```

Comment:
As you have missed PR  #17815 going through, can you please move these parameters to your info.json? Please reference the PR for how this is done.

Reply:
the changes have been made, thank you!